auto3dseg/algorithm_templates/dints/configs/hyper_parameters.yaml

```diff
@@ -5,31 +5,42 @@ data_file_base_dir: null
 data_list_file_path: null
 fold: 0
 
+transforms:
```

Comment:
I saw the these changes reverted my previous bug-fix

https://github.com/Project-MONAI/research-contributions/pull/224

The PR contains the details why it could be problem to define `transforms` and `cache_rate` here.
@dongyang0122 can you check if we can avoid getting similar bugs? Thanks!

Reply:
I have restored the settings to the previous state.